src/Translatable/Query/TreeWalker/TranslationWalker.php

```diff
@@ -460,8 +461,11 @@ private function getCastedForeignKey($component, $typeFK, $typePK)
             return $component;
         }
 
-        // try to look at postgres casting
-        if ($this->platform instanceof PostgreSqlPlatform) {
+        // doctrine/dbal 3.x drops support for PostgreSQL 9.3 and earlier, the below check prefers the
+        // `PostgreSQL94Platform` class as it is available in both major DBAL versions and falls back to the
+        // deprecated `PostgreSqlPlatform` class to account for older PostgreSQL versions with DBAL 2.x.
```

Comment:
Note that DBAL 3.2 reintroduces the `PostgreSqlPlatform` class as the base class for postgresql platforms (still supporting only PG 9.4+)

Reply:
Well, ain't that something.  Either way, the second `instanceof` check is going to be needed for DBAL 3.1 I guess.